src/rules/no-space-in-emphasis.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * @fileoverview Rule to prevent spaces around emphasis markers in Markdown.
+ * @author Pixel998
+ */
+
+//-----------------------------------------------------------------------------
+// Imports
+//-----------------------------------------------------------------------------
+
+import { findOffsets } from "../util.js";
+
+//-----------------------------------------------------------------------------
+// Type Definitions
+//-----------------------------------------------------------------------------
+
+/**
+ * @typedef {import("../types.ts").MarkdownRuleDefinition<{ RuleOptions: []; }>}
+ * NoSpaceInEmphasisRuleDefinition
+ */
```

Comment:
```suggestion
/**
 * @typedef {import("../types.ts").MarkdownRuleDefinition<{ RuleOptions: []; }>}
 * NoSpaceInEmphasisRuleDefinition
 */
```

Could you merge the `main` branch and update the code to use the `@import` JSDoc syntax as shown in the ongoing changes in https://github.com/eslint/markdown/pull/367/files?

You can refer to the following image as a guide:

![image](https://github.com/user-attachments/assets/1313f60d-07fa-422a-92f9-9c5c2b506fe1)